group_vars/all

```diff
@@ -48,7 +53,6 @@ galaxy_config:
     database_connection: "{{ galaxy_db }}"
     tool_dependency_dir: "{{ tool_dependency_dir }}"
     ftp_upload_dir: "{{ proftpd_files_dir }}"
-    ftp_upload_site: ftp://{{ galaxy_hostname }}
```

Comment:
Should not be keeping this line with something like

`ftp_upload_site: ftp://$IP_ADDRESS`

??